hedera-node/hedera-mono-service/src/test/java/com/hedera/test/serde/EqualityType.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.test.serde;
+
+public enum EqualityType {
```

Comment:
Some javadoc explaining how this enum is meant to be used would be helpful

Reply:
@tinker-michaelj can you add it please?

Reply:
@mhess-swl please check it up